studio/schemas/deskStructure.ts

```diff
@@ -22,13 +23,26 @@ export default (S: StructureBuilder) =>
     .title("Content")
     .items([
       S.listItem()
-        .title("Company Information")
+        .title("Company Details")
         .icon(CaseIcon)
         .child(
-          S.document()
-            .schemaType(companyInfoID)
-            .documentId(companyInfoID)
-            .title("Company Information"),
+          S.list()
+            .title("Company Details")
+            .items([
+              S.listItem()
+                .title("Company Information")
```

Comment:
I think it's better to keep the same title Company Details -> Company Details

Reply:
Even though the parent company details have two children? I was just thinking that it was better to seperate it to 
- company detail
-- company information
-- offices (now locations)

and not 
- company details
-- company details
-- offices (now locations)

Reply:
Oooh. sorry! Yes, I totally agree 👍